packages/stream-server-uws/README.md

```diff
@@ -0,0 +1,41 @@
+# Meteor's DDP stream server based on uWS
+
+## Installation
+
+1. Install the fork of the uWebSocket Node.js bindings:
+
+    ```bash
+    meteor npm install sc-uws
+    ```
+    
+2. Pin the older version of the `socket-stream-client` package,
+for using native WebSocket instead of SockJS:
+
+    ```bash
+    meteor add socket-stream-client@=0.2.1
+    ```
+
+3. Install the `stream-server-uws` package:
+
+    ```bash
+    meteor add stream-server-uws
+    ```
+
+## Activation
+
+In the `.meteor/packages` place the package `stream-server-uws` before packages:
```

Comment:
You should be able to skip this recommendation if you put
```js
api.use("stream-server-uws", { weak: true });
```
in the `package.js` files for these modules, so that `stream-server-uws` will load before them (but only if it is installed).

Reply:
Yes, I could write this package like this:

`ddp-server/package.js`

```js
Package.onUse(function (api) {
  api.use("stream-server-uws", { weak: true });
}
```

`ddp-server/livedata.js`

```js 
  var StreamServer = StreamServerUWS || StreamServer;
  self.stream_server = new StreamServer;
```

But in this case we don't provide our users the simple way of changing of the stream server.

So I create the `StreamServers` array, and write guide how to extend it right way.